### PR TITLE
Created event_type and event models

### PR DESCRIPTION
**Pull request by noobbyte**

### CODE REVIEW

**Comment by Jma353** [comment]
2 space indentation :P

File: src/app/events/models/event.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+from . import *
+
+class Event(Base):
+    __tablename__ = 'events'
+
+    id = db.Column(db.Integer, primary_key=True)
+    event_type_id = db.Column(db.Integer, db.ForeignKey('event_type.id'))
+    event_type = db.relationship('EventType', backref='events')
```
Comment (Jma353) [inline src/app/events/models/event.py]: We usually define fields, then ORM driven relationships below, so move this down.

File: src/app/events/models/event.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    payload = db.Column(db.JSON)
+
+    def __init__(self, **kwargs):
+        self.event_type_id = kwargs.get(event_type_id)
```
Comment (Jma353) [inline src/app/events/models/event.py]: `kwargs.get('event_type_id')`

Does this run?

File: src/app/events/models/event.py [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+        self.payload = kwargs.get(payload)
```
Comment (Jma353) [inline src/app/events/models/event.py]: `kwargs.get('payload')`

File: src/app/events/models/event_type.py [hunk 4]
```diff
@@ -0,0 +1,17 @@
+from . import *
+
+class EventType(Base):
+    __tablename__ = 'event_types'
+
+    id = db.Column(db.Integer, primary_key=True)
+    name = db.Column(db.String(50))
+    creator = db.Column(db.String(50))
```
Comment (Jma353) [inline src/app/events/models/event_type.py]: Add a `# TODO` here, b/c @young-k made us a `user` model :D

**Comment by Jma353** [comment]
A suggestion for testing these changes, migrate the db + screenshot the resultant schema. You can also import the flask app into the python repl and manually create models to make sure they are being stored properly.

**Comment by noobbyte** [comment]
migrated

**Comment by Jma353** [comment]
Looks great!

**Pull request merged**